doc/releases/migration-guide-4.2.rst

```diff
@@ -500,6 +500,9 @@ OpenThread
 SPI
 ===
 
+* Renamed Renamed :kconfig:option:`CONFIG_SPI_MCUX_LPSPI` to :kconfig:option:`CONFIG_SPI_NXP_LPSPI`,
```

Comment:
```suggestion
* Renamed :kconfig:option:`CONFIG_SPI_MCUX_LPSPI` to :kconfig:option:`CONFIG_SPI_NXP_LPSPI`,
```

Reply:
no, this is valid grammar, it's saying it's renaming the previously renamed config to another name. (just kidding, its a typo)

Reply:
> no, this is valid grammar, it's saying it's renaming the previously renamed config to another name. (just kidding, its a typo)

ROFL - you actually got me there :)